code/processes/dqe.q

```diff
@@ -6,19 +6,26 @@ partitiontype:@[value;`partitiontype;`date];
 getpartition:@[value;`getpartition;
   {{@[value;`.dqe.currentpartition;
     (`date^partitiontype)$(.z.D,.z.d)gmttime]}}];
+writedownperiodengine:@[value;`writedownperiodengine;0D01:00:00];
 
 configcsv:@[value;`.dqe.configcsv;first .proc.getconfigfile["dqengineconfig.csv"]];
 resultstab:([]procs:`$();funct:`$();table:`$();column:`$();resvalue:`long$());
 
 init:{
   .lg.o[`init;"searching for servers"];
   .servers.startup[];                                                                                           /- Open connection to discovery
+  .timer.once[.eodtime.nextroll;(`.u.end;.dqe.getpartition[]);"Running EOD on Engine"];                         /- set timer to call EOD
```

Comment:
should this be `.timer.once` ? doesn't that mean it'll only happen at the first EOD & not every day after that? or is this `init` function called again or something? (if so, I'd maybe add a comment to the top of this function to point out it gets called at every EOD so others aren't potentially confused by this down the line)

Reply:
Yes, this is .timer.once - scheduled once at EOD. To have it running every day, init is called again in .u.end, so we will add comment on top of init